controllers/admin/AdminLegacyLayoutController.php

```diff
@@ -151,7 +151,7 @@ public function initContent()
         $isProductPage = ('AdminProducts' === $this->controller_name);
 
         $vars = [
-            'viewport_scale' => $isProductPage ? '0.75' : '1',
+            'viewport_scale' => '1',
```

Comment:
I think you can delete the `$isProductPage` variable. What about setting viewport_scale directly in the template instead of using a smarty variable?